.gitignore

```diff
@@ -29,5 +29,8 @@ node_modules
 # Jetbrains IDEs
 .idea
 
+# vim swp files
```

Comment:
I actually took a note from the above .idea line and thought it would be fine to add it in, will remove